OrbitQt/orbitmainwindow.cpp

```diff
@@ -72,8 +72,22 @@ OrbitMainWindow::OrbitMainWindow(QApplication* a_App,
 
   GOrbitApp->AddCaptureStartedCallback(
       [this] { ui->HomeTab->setDisabled(true); });
-  GOrbitApp->AddCaptureStoppedCallback(
-      [this] { ui->HomeTab->setDisabled(false); });
+
+  auto finalizing_capture_message_box = std::make_shared<QMessageBox>(this);
```

Comment:
Since you were talking about the QMessageBox in the standup:

I think you can use QProgessDialog here. This dialog has some advantages. It only shows up after a certain period of time. So when finalization is short, you don't have a glitch. By default it has a progress bar and a cancel button, but you can disable the cancel button and you can set the progress bar to undetermined state by setting a range of (0,0).

Reply:
That sound very promising, I'll have a look, thanks!

Reply:
It took me a while to get more or less the look I wanted both on Linux and Windows but I think it's fine now. It seems that even if the cancel and close buttons are disabled, it's still possible to dismiss the dialog by pressing Esc. It is possible to work around it, but thinking about it, after having been notified that more capture data is still coming, the user should still be able to interact with the user interface in the meantime (they can't do a lot at the moment, but still), so I also kept the cancel button. We just need to make sure to keep the actions that the ui allows consistent with this state of data still coming.

Reply:
Sounds good for me. But shouldn't we rename the "cancel" button. We are not canceling the receiving of new events, right? We rather would have something like "close" the dialog, or "acknowledge" the message. Even "OK" would be less missleading.

Reply:
It's called cancel button by Qt code, but I originally went for "OK"? Why do you think it's misleading?

Reply:
Sorry. I think "OK" would be fine. 
"Cancel" instead sounds like you want to cancel "finalization of the capture".

Reply:
Can you rename it?

Reply:
It is renamed, it's the second parameter of the constructor.

Reply:
Ah now I see. Sorry. Yes LGTM. Thanks